src/templates/activityTemplate/suggestionsSection/suggestions.tsx

```diff
@@ -2,18 +2,21 @@ import React, { useState, useEffect, useLayoutEffect, useRef } from 'react';
 import NewReplyForm from './newReplyForm';
 import { CommonSuggestionFormProps, Error } from './index';
 import { useQueryParam, StringParam, NumberParam } from 'use-query-params';
-import { parseDate, prependApiUrl } from '../../../utils/helpers';
+import { parseDate, parseLinkUrl, prependApiUrl } from '../../../utils/helpers';
 import AttachmentIcon from '../../../images/attachment.inline.svg';
 import LinkIcon from '../../../images/link.inline.svg';
+import LikeIcon from '../../../images/like.inline.svg';
 import PinnedIcon from '../../../images/pinned.inline.svg';
 import TimeIcon from '../../../images/time.inline.svg';
 import { sendSuggestionLike, sendSuggestionUnlike } from '../../../services/activity';
 import { v4 as uuidv4 } from 'uuid';
 import toast from 'react-hot-toast';
 import clsx from 'clsx';
+import { linkSync } from 'fs';
 import { hexToRgba } from '../../../utils/color';
 import Pill from '../../../components/pill';
 import { useTranslation } from 'react-i18next';
+import * as _ from 'lodash';
```

Comment:
Maybe just `import debounce from 'lodash.debounce`, don't need to bundle the whole thing.